tutorials/introduction.md

```diff
@@ -5,18 +5,37 @@ slug: /token-gating-tutorial
 ---
 
 # Collab.Land Token Gating Tutorial
+
 ## Overview
+
 This guide walks you through creating your own token gate for your website using the Collab.Land API. This enables you to restrict access to elements of or pages of your site based on ownership of certain blockchain assets.
 
 ### Website Gating Logic Flow
+
 View logic diagram [on whimsical](https://whimsical.com/embed/S5jqcu5XzqBrnRtvgrPBj1@2Ux7TurymNLRX4obG1NB).
 
-This tutorial implements gating logic by calling Collab.Land API with a user's wallet address. The API checks against the website-defined ⚠ link needed! [Token-Gating Role] definitions and returns `True/False` if the address fits the provided criteria or not.
+This tutorial implements gating logic by calling Collab.Land API [`access-control/check-roles`](../docs/downstream-integrations/api/token-gating.md) with a user's wallet address and [token gating rules](#token-grating-rule) as the payload and returns `true/false` for each rule to indicate if the given address fits the provided criteria or not.
```

Comment:
@agnes512 these `downstream-integrations` references are broken in the final site. What were you intending to reference?